conans/client/recorder/action_recorder.py

```diff
@@ -32,6 +32,10 @@ def _cpp_info_to_dict(cpp_info):
             doc["configs"] = configs_data
             continue
 
+        if it == "_rootpath":
```

Comment:
This hack is because the json of the action recorder now receives the "rootpath" and I made it private to control the propagation of the rootpath to the components when it is changed (in the setter), So I needed to "hack" it to not break.